### DISCUSSION TITLE
How to make sure that an administrator doesn't pull the carpet from his/her own feet

### DISCUSSION THREAD

**Discussion by wintermeyer** [3Stones-io/omedis#333] [General]
We should discuss if it makes sense to hard code into the `users` policies that an administrator can delete his/her own user account. I am for that. Otherwise it would be easy to lock oneself out of the system by accident.

**Comment by fradarr** [comment]
This and soft deletion for "everything" as well as proper logging of all activities (not the resource name) /actions on the app.
